src/App.tsx

```diff
@@ -1,39 +1,63 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './Components/Clock/Clock';
+
+interface State {
+  hasClock: boolean,
+  clockName: number,
+}
 
 function getRandomName(): string {
   const value = Date.now().toString().slice(-4);
 
-  return `Clock-${value}`;
+  return value;
 }
 
-export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
+export class App extends React.Component<{}, State> {
+  state: State = {
+    hasClock: true,
+    clockName: 0,
+  };
+
+  timerId = 0;
+
+  componentDidMount(): void {
+    this.timerId = window.setInterval(() => {
+      this.setState({ clockName: +getRandomName() });
```

Comment:
It is better to transfer the conversion to the number (+) type in the middle of the helper function.